README.md

```diff
@@ -23,6 +23,22 @@ var refmintClient = new Refmint({
 &emsp;baseUrl: BaseURLOptions.TESTNET<br />
 });
 
+Update note: As of v.1.1.24, The Game and NFT project clients have been seperated. They are now seperate and can be initiated as below:
```

Comment:
Please update this to 1.2.0 for big breaking change. and then update the docs as well later today